capstone/client/src/components/ListPage.js

```diff
@@ -184,12 +184,12 @@ class ListPage extends Component {
       this.setState({
         errorMessage: "There was an error saving your list.",
       })
-    });   
+    });
   }
 
   getItemTypes = () => {
     axios.get(
-      '/api/v1/get-item-types?page=0'
+      '/api/v1/get-item-types?page=0' 
```

Comment:
Better to use params here: 
https://kapeli.com/cheat_sheets/Axios.docset/Contents/Resources/Documents/index